common/beerocks/tlvf/CMakeLists.txt

```diff
@@ -3,7 +3,13 @@ cmake_minimum_required (VERSION 2.8.12)
 set(PROJECT_NAME btlvf)
 project(${PROJECT_NAME})
 
-find_package(PythonInterp REQUIRED)
+# TODO: FIXME: For some reason CMake detects only the Python from the cross-toolchain
```

Comment:
The problem is with the pyYAML library dependency. Not all cross-toolchains contain this package.
As for this specific hack, it may be an issue with the older Turris tree (3.x), perhaps it doesn't exist in the latest one.
I suggest opening an issue and handling it separably from this PR.

Reply:
Can't we quickly check if this is needed in the new Turris tree? I think @arnout or @rmelotte can help with a quick check.
Of the answer is that it is needed for now, then @morantr please open an issue on that and add the issue link in the TODO comment.

Reply:
I've reverted this bit on the dev/turris-omnia-support branch and it built fine.

Anyway it doesn't make any sense that this would be needed in common but not in framework.

Therefore, please don't include this change.